functions/helpers.bats

```diff
@@ -2,6 +2,24 @@
 
 load helpers
 
+testthis() {
+  tryUntil "wget -S --spider -t 3 --waitretry=4 $1 2>&1 | grep -q 'HTTP/1.1 200 OK'" 5 2
+  ret=$(($? - $2))
+  if [ $ret -ne 0 ]; then echo "OK"; return 0; else echo "failed."; return 1; fi
+}
+
+@test "unit-tryExistingSite" {
+  testthis "http://www.openhab.org/" 5
+  [ "$status" -eq 0 ]
+  echo -e "# \n\e[32mGetting www.openhab.org in tryUntil loop succeeded." >&3
+}
+
+@test "unit-tryNonExistingSite" {
+  run testthis "http://nothisdoesnotexistforsure/" 0
+  [ "$status" -eq 0 ]
```

Comment:
Its strange to have status "0" during a failure. 
We need to think twice about this. I will give it some effort in the end of the week.

Reply:
It's now the number-of-attempts-left which is what tryUntil returns.